### ISSUE TITLE
background image of HbbTV version

### ISSUE THREAD

**Issue by ffallahi6303** [DVBproject/DVB-I-Reference-Client#29]
I think there is a problem in[https://github.com/DVBproject/DVB-I-Reference-Client/blob/master/frontend/hbbtv/CommonUI/bg_menu.png](url) . 

this is page that I see in hbbtv client. service list can't be displayed. also background image is not fit.

![image](https://user-images.githubusercontent.com/44312423/93012261-198fe300-f5b4-11ea-8fe3-7ca5f544cf3b.png)

. I don't have this problem in android client. I added my service list address in dvbi-common.js and sleper master and related files...

**Comment by paulhiggs** [comment]
On the size/scale of the background image for the service list presentation, we should add `background-size: cover` to `#wrapper` in `frontend\hbbtv\launcher\navi.css`

On the service list not being presented, have you validated your service list against the A177 schema? You can send me the service list URI and edited dvbi-common.js and I can check it for you

**Comment by ffallahi6303** [comment]
Hi, I will send it. actually finally I used an old version of client and every thing was ok about it,'but some thing is wrong about this new version of client that is available in the github, now. now I uploaded this client with its service list.. and android and HbbTV version can't show service provider selection... please check the code that is available in github

**Comment by ffallahi6303** [comment]
I solved my problem.

**Comment by ffallahi6303** [comment]
The problem was that I had replaced old version of slepr-master.xml and DVBI-common.js and index.php in this new version of code.( these files had my service list information and I had edited them before) . It seems these files had some changes in new version of code but I didn't understand.  
when I used new version of these files and edited them again for addressing my service list, the problem got solved.

the issue is that in new code change, I didn't see any mention to change in these files then I made mistake.
